add BFloat16 support for AvgPool2d on CPU

mingfeima opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #61015
* #58297
* __->__ #66927
* #66929

Differential Revision: [D33353198](https://our.internmc.facebook.com/intern/diff/D33353198)

mingfeima committed:
add BFloat16 support for AvgPool2d on CPU

[ghstack-poisoned]

mingfeima committed:
Update on " add BFloat16 support for AvgPool2d on CPU"




[ghstack-poisoned]

mingfeima committed:
Update on " add BFloat16 support for AvgPool2d on CPU"




[ghstack-poisoned]

mingfeima committed:
Update on " add BFloat16 support for AvgPool2d on CPU"




[ghstack-poisoned]

mingfeima committed:
Update on " add BFloat16 support for AvgPool2d on CPU"




[ghstack-poisoned]

mingfeima committed:
Update on " add BFloat16 support for AvgPool2d on CPU"




[ghstack-poisoned]

VitalyFedyunin commented:
@VitalyFedyunin has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D33353198).

mingfeima committed:
Update on " add BFloat16 support for AvgPool2d on CPU"


Differential Revision: [D33353198](https://our.internmc.facebook.com/intern/diff/D33353198)

[ghstack-poisoned]

VitalyFedyunin commented:
@VitalyFedyunin has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D33353198).

mingfeima committed:
Update on " add BFloat16 support for AvgPool2d on CPU"


Differential Revision: [D33353198](https://our.internmc.facebook.com/intern/diff/D33353198)

[ghstack-poisoned]

mingfeima committed:
Update on " add BFloat16 support for AvgPool2d on CPU"


Differential Revision: [D33353198](https://our.internmc.facebook.com/intern/diff/D33353198)

[ghstack-poisoned]

VitalyFedyunin commented:
@VitalyFedyunin has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D33353198).
